hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1046,6 +1046,14 @@
       en_run_modes: ["strap_tests_mode"]
       run_opts: ["+lc_at_prod=1"]
     }
+    {
+      name: chip_sw_rv_core_ibex_address_translation
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["sw/device/tests/rv_core_ibex_address_translation_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      // Timeout based on a ~7 minute dvsim runtime.
```

Comment:
FYI, we are adding a wall-clock timeout capability to dvsim, in https://github.com/lowRISC/opentitan/pull/13865, with which you can turn this into "timeout_mins: 7"

Reply:
Oooo nice and its now merged! CI has caught a bazel lint error to fix; I'll look at using `timeout_mins` while I'm at it.